packages/affiliates/libs/contracts.js

```diff
@@ -1,7 +1,8 @@
 const ethers = require("ethers");
```

Comment:
If we using web3.js and `getAbi` from `@uma/core` then I feel that we should not use `ethers` at all. I don't like muddling using the one library for somethings and the other for other things.

Reply:
yes theres a task to remove ethers, id like to do that as a seperate issue https://github.com/UMAprotocol/protocol/projects/6#card-48326707